layouts/partials/contact_foot_en.html

```diff
@@ -0,0 +1,41 @@
+<br />
+<br />
+    <div itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
+        <div class="halfPanel">
+		 <h2>E-mail Contacts</h2>
+
+        <dl>
+                <dt>General information</dt>
+                <dd><a href="mailto:info@enectiva.cz" itemprop="email">info@enectiva.cz</a></dd>
+
+                <dt>Technical support</dt>
+                <dd><a href="mailto:podpora@enectiva.cz" itemprop="email">podpora@enectiva.cz</a></dd>
+
+                <dt>Business issue</dt>
+                <dd><a href="mailto:obchod@enectiva.cz" itemprop="email">obchod@enectiva.cz</a></dd>
+
+                <dt>Bussiness partnership</dt>
+                <dd><a href="mailto:partner@enectiva.cz" itemprop="email">partner@enectiva.cz</a></dd>
+
+        </dl>
+        </div>
+
+        <div class="halfPanel">
+          <h2>Phone Contacts</h2>
+          <ul>
+            <li>Phone : <span itemprop="telephone">+420 222 766 950</span></li>
+            <li>Phone : <span itemprop="telephone">+420 222 766 951</span></li>
+            <li>Mobile : <span itemprop="telephone">+420 732 916 385</span></li>
+            <li>Fax : <span itemprop="faxNumber">+420 222 766 959</span></li>
```

Comment:
I think that any language use space in front of semicolons. It's possible that normal format will be something like:
**Phone:**
In my language, all paragraphs starts with uppercase but, if you want to start with lowercase it will be like this.